src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryDate.cs

```diff
@@ -328,6 +328,12 @@ public static FormulaValue Date(IRContext irContext, NumberValue[] args)
             var month = (int)args[1].Value;
             var day = (int)args[2].Value;
 
+            // dates before 1900 get converted to post-1900
```

Comment:
>// [](http://example.com/codeflow?start=12&length=2)

We had a interpreter test for this. 
Easier to just fix this bug than skip the test.